wallet/src/ui/app/components/network-switch/Network.module.scss

```diff
@@ -0,0 +1,111 @@
+.network {
```

Comment:
The switch seems a bit too big, especially in the popup view. We could make it a bit smaller less margins etc.

<img width="369" alt="Screenshot 2022-06-13 at 11 22 31" src="https://user-images.githubusercontent.com/10210143/173333499-a74b7cc0-9cd5-4c5e-aed7-482cf5ededf0.png">

Reply:
I will make it smaller.  Developed it on the browser version